verification/cbmc/proofs/aws_byte_buf_write_be16/cbmc-batch.yaml

```diff
@@ -0,0 +1,8 @@
+build_memory: 32000
+cbmcflags: "--unwind;1;--unwindset;memcpy_impl.0:11;--object-bits;8;--pointer-primitive-check;--bounds-check;--conversion-check;--div-by-zero-check;--float-overflow-check;--nan-check;--pointer-check;--pointer-overflow-check;--signed-overflow-check;--undefined-shift-check;--unsigned-overflow-check;--unwinding-assertions"
+coverage_memory: 32000
+expected: SUCCESSFUL
+goto: /Users/saspadhi/Repos/aws-c-common/verification/cbmc/proofs/aws_byte_buf_write_be16/gotos/aws_byte_buf_write_be16_harness.c.goto
+jobos: ubuntu16
+property_memory: 32000
+report_memory: 32000
```

Comment:
@karkhaz Isn't these files generated automatically now?

Reply:
As mentioned offline, I recommend to commit syntactically-incorrect cbmc-batch.yaml files.

Reply:
@SaswatPadhi could you replace these `cbmc-batch.yaml` files with the following message?
```c
:
  This file marks this directory as containing a CBMC proof. This file
  is automatically clobbered in CI and replaced with parameters for
  running the proof.
```

Reply:
I was looking at the other proof directories, those that are not modified in this PR, and they don't seem to have this message. Should we make a separate PR and replace all of them?

Reply:
Yes, let's not pollute this PR with unnecessary changes. We can update the others on a separate one.

Reply:
Just pushed all the updated yaml files.